app/helpers/preview_helper.rb

```diff
@@ -1,8 +1,5 @@
 module PreviewHelper
-  def preview_data_attrs(show_preview = true, preview_type, id, target)
-    if show_preview
-      attrs = "data-behavior=\"#{preview_type}\" data-preview-url=\"#{preview_path(id)}\" data-preview-target=\"#{target}\""
-    end
-    (attrs ||= '').html_safe
+  def preview_data_attrs(preview_type, id, target)
```

Comment:
It may be helpful orientation to mention in a comment here that the possible values for `data-behavior` passed through `preview_type` (`preview-gallery`, `preview-filmstrip`, and `preview-brief`) are added so that javascript can pick them, and that there are corresponding javascript files such as `jquery.preview-filmstrip.js` where this happens.

Reply:
That is being removed in another PR (https://github.com/sul-dlss/SearchWorks/pull/3803) , so I'm not sure it's worthwhile